File: 프로그래머스/0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.py
def solution(my_str, n):
    answer = []
    temp = ""
    
    for i in my_str:
        temp += i
        if len(temp) == n:
            answer.append(temp)
            temp = ""
            
    if len(temp) < n and len(temp) != 0:
        answer.append(temp)
    
    return answer

File: 프로그래머스/0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.swift
import Foundation

func solution(_ my_str:String, _ n:Int) -> [String] {
    var answer: [String] = []
    var temp: String = ""

    for i in my_str {
        temp.append(i)
        
        if temp.count == n {
            answer.append(temp)
            temp = ""
        }
    }
    
    if temp.count != 0 && temp.count < n {
        answer.append(temp)
    }
    
    return answer
}